src/library/simplified/extension/doubleclick-src-type-cat-HSR.js

```diff
@@ -1,4 +1,4 @@
-if(b.siteId && utag.isHSR() ){
+if(b.siteId && utag.isHSR() || utag.isPSR() ){
```

Comment:
can we have it like this in case b.siteId doesn't exist for PSR page?
if((b.siteId && utag.isHSR()) || (b.siteId && utag.isPSR()) )